chore(deps): migrate contracts to @lidofinance/lsv-cli 1.8

## Context

The dependency maintenance pass attempted to bump `@lidofinance/lsv-cli` from `1.4.0` to `1.8.0` in the `contracts` workspace.

The bump is not immediately safe because `pnpm -F contracts run build` fails while loading Hardhat config:

```text
Error: Cannot find module '@lidofinance/lsv-cli/dist/utils/fetchCL'
Require stack:
- contracts/scripts/operational/yieldBoost/testing/unstakePermissionless.ts
- contracts/hardhat.config.ts
```

## Expected work

Update the yield boost test/operational scripts to use the supported API or replacement path in `@lidofinance/lsv-cli` 1.8+, then bump the dependency.

## Validation

- `pnpm install --frozen-lockfile`
- `pnpm -F contracts run build`
- Smart contract CI test job

## Notes

The dependency PR kept `@lidofinance/lsv-cli` on `1.4.0` to avoid breaking the current dependency refresh.

--- a/contracts/package.json
+++ b/contracts/package.json
@@ -28,7 +28,7 @@
     "@chainsafe/persistent-merkle-tree": "1.2.5",
     "@consensys/eslint-config": "workspace:*",
     "@ethereumjs/util": "10.1.1",
-    "@lidofinance/lsv-cli": "1.4.0",
+    "@lidofinance/lsv-cli": "1.8.0",
     "@lodestar/types": "1.42.0",
     "@nomicfoundation/hardhat-ethers": "3.1.3",
     "@nomicfoundation/hardhat-foundry": "1.2.1",

--- a/contracts/scripts/operational/yieldBoost/testing/unstakePermissionless.ts
+++ b/contracts/scripts/operational/yieldBoost/testing/unstakePermissionless.ts
@@ -2,7 +2,7 @@ import {
   fetchBeaconHeader,
   fetchBeaconState,
   fetchBeaconHeaderByParentRoot,
-} from "@lidofinance/lsv-cli/dist/utils/fetchCL";
+} from "@lidofinance/lsv-cli/dist/utils/fetch-cl.js";
 import { createBeaconHeaderProof, createStateProof } from "@lidofinance/lsv-cli/dist/utils/proof/proofs.js";
 import { hexlify, AbiCoder } from "ethers";
 import * as fs from "fs";

